tests/Tests/ORM/Functional/SchemaValidatorTest.php

```diff
@@ -52,6 +53,18 @@ public static function dataValidateModelSets(): array
         $modelSets = [];
 
         foreach (array_keys(self::$modelSets) as $modelSet) {
+
+            /**
+             * GH11608 Tests whether the Schema validator picks up on invalid mapping. The entities are intentionally
+             * invalid, and so for the purpose of this test case, those entities should be ignored.
```

Comment:
Then I'd avoid creating a model set entirely. There are plenty of tests that don't.